code/datums/components/speed_curse.dm

```diff
@@ -0,0 +1,38 @@
+/datum/component/speed_curse
+    var/mob/holder
```

Comment:
This var should be nulled in destroy if you actually needed the var for this component